_includes/releases/v22.2/v22.2.1.md

```diff
@@ -0,0 +1,386 @@
+- This patch fixes an issue with the `enforce_home_region` session setting, which may cause [`SHOW CREATE TABLE`](../v22.2/show-create.html) or other non-DML statements to error out if the optimizer plan for the statement involves accessing a multi-region table. [#90204][#90204]
+- Fixed a bug that could potentially causing changefeeds with [`initial_scan_only`](../v22.2/create-changefeed.html#initial-scan) to missing messages. Now, the changefeed ensures that all messages have successfully flushed to the sink prior to completion. [#90277][#90277]
+- Now, during [JWT-based authentication](../v22.2/sso-sql.html), algorithm type is inferred if it is not specified by the JWKS. This enables support for a wider range of JWKS. [#90289][#90289]
+- Fixed a bug that caused incorrect evaluation of comparison expressions involving time and interval types, like `col::TIME + '10 hrs'::INTERVAL' > '01:00'::TIME`. [#90368][#90368]
+- This patch fixes detection and erroring out of queries using a locality-optimized join when session setting `enforce_home_region` is `true` and the input table to the join has no home region or its home region does not match the gateway region. [#90199][#90199]
+- A bug introduced in v22.1.9 that caused nodes to refuse to run jobs under rare circumstances is fixed. [#90271][#90271]
+- Fixed the calculation of the `pg_attribute.attnum` column for indexes so that the `attnum` is always based on the order the column appears in the index. Also fixed the `pg_attribute` table so that it includes stored columns in secondary indexes. [#90458][#90458]
+- Previously, during [restore](../v22.2/restore.html) planning, the restoring cluster's codec was accidentally used to reason about spans in the [backup manifest](../v22.2/backup-architecture.html). When a backup was restored by a different tenant, two bugs described in [#90475](https://github.com/cockroachdb/cockroach/issues/90475) and [#90474](https://github.com/cockroachdb/cockroach/issues/90474) could occur. This patch fixes these bugs. [#90527][#90527]
+- TTL decoding error messages now correctly contain hex-encoded key bytes instead of hex-encoded key pretty print output. [#90723][#90723]
+- When running Cockroach inside of a Docker container on macOS, when mounting a host filesystem into the container, the total available capacity calculation of the filesystem could be reported incorrectly. This is now fixed. [#90873][#90873]
+- Fixed a bug that caused incorrect results and internal errors when a `LEFT JOIN` operated on a table with [virtual computed columns](../v22.2/computed-columns.html). The bug only presented when the optimizer planned a "paired joiner". Only values of the virtual columns would be incorrect—they could be be `NULL` when their correct value was not `NULL`. An internal error would occur in the same situation if the virtual column had a `NOT NULL` constraint. This bug was present since version v22.1.0. [#90997][#90997]
```

Comment:
Suggest:

> ... would be incorrect - they could be `NOT NULL` when their correct value was not `NULL`.

Reply:
Oops! I typod my suggestion, sorry! Should have been:

> ... would be incorrect - they could be `NULL` when their correct value was `NOT NULL`.

Feel free to disregard.